tests/integration/replication.tcl

```diff
@@ -1351,6 +1351,13 @@ start_server {tags {"repl" "external:skip"}} {
         waitForBgsave r
     }
 
+    test "Trigger a successful SAVE and normalize the lastbgsave_status state" {
```

Comment:
i'd suggest to drop the `test` proc.
we don't test anything, and there's no need to print "OK" to the console.

btw, looking at that kill, i wonder why it was needed, the fork child will get terminated anyway when the replica drops.
and then, being terminated naturally, it wouldn't affect the PING, right?

btw, another way to prevent it from affecting PING is to use diskless, right?

looking at the various alternatives, i think dropping the kill is the best one, but i wonder if i'm missing anything?

Reply:
> btw, looking at that kill, i wonder why it was needed, the fork child will get terminated anyway when the replica drops.
and then, being terminated naturally, it wouldn't affect the PING, right?

yes, in diskless way, when the replica was dropped, we will do a kill SIGUSR1, this way dosen't set the lastbgsave_status to C_ERR. but in disk way, we won't kill the child.

> btw, another way to prevent it from affecting PING is to use diskless, right?

yes

> looking at the various alternatives, i think dropping the kill is the best one, but i wonder if i'm missing anything?

since we do have the `waitForBgsave`, so i guess i will just drop the kill, and let `waitForBgsave` take care of the bgsave